pkg/apache/apache_test.go

```diff
@@ -6,25 +6,25 @@ import (
 
 func testEq(a, b []string) bool {
 
-    if a == nil && b == nil {
-        return true;
-    }
+	if a == nil && b == nil {
+		return true
+	}
 
-    if a == nil || b == nil {
-        return false;
-    }
+	if a == nil || b == nil {
+		return false
+	}
 
-    if len(a) != len(b) {
-        return false
-    }
+	if len(a) != len(b) {
+		return false
+	}
 
-    for i := range a {
-        if a[i] != b[i] {
-            return false
-        }
-    }
+	for i := range a {
+		if a[i] != b[i] {
+			return false
```

Comment:
Could you explain the change to tabs instead of spaces?

Reply:
These issues were done by executing `go fmt`, and the changes in the dependency ordering were reported by the `staticcheck` linter.